node-wasm/js/worker.js

```diff
@@ -1,10 +1,8 @@
-import init, { NodeWorker } from "lumina-node-wasm"
+import { NodeWorker } from "lumina-node-wasm"
 
 Error.stackTraceLimit = 99;
 
-init().then(async () => {
-  let worker = new NodeWorker(self);
-  console.log("Starting NodeWorker");
+let worker = new NodeWorker(self);
+console.log("starting worker: ", worker);
```

Comment:
can you revert the message change?
```suggestion
console.log("Starting NodeWorker");
```